contrib/builder/rpm/s390x/clefos-base-s390x-7/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/s390x/generate.sh"!
+#
+
+FROM sinenomine/clefos-base-s390x
```

Comment:
This isn't exactly official -- is ClefOS something we want to support? (appears to be a Z-specific CentOS/RHEL derivative?)